tools/github/data/common-labels.csv

```diff
@@ -228,6 +228,7 @@ VideoAnalyzer,Azure Video Analyzer,e99695
 Visual Studio,,e99695
 Web Apps,,e99695
 WebPubSub,,e99695
+auto-close-exempt,Prevents the auto-close from closing based on max lifetime,bbbbbb
```

Comment:
@RickWinter, I'm going to defer to @jsquire and @ronniegeraghty on the color of this label. I'm sure one or both will want a different color.

Reply:
I think the color is fine. It's very grey and dull, and I don't really want to bring any attention to this label. I want it to be a tool only the SDK team uses so an unassuming color works here.

Reply:
I'm going to ask that we adjust to match the other greyed out labels, just for consistency.  That's the form that I've pushed out to the other repos.